### ISSUE TITLE
[Request] Default monospace  & code colors in dark mode are too bright?

### ISSUE THREAD

**Issue by fortunewalla** [mohzy83/NppMarkdownPanel#71]
In dark mode, the default code & monospace colors are too bright for dark mode. Can you make the default code colors the same as general text dark mode colors?

Yes, we can tweak the CSS file but the generated HTML file uses the DIV color names 'black' & 'white' which are hardcoded.

Example:

```html
<div id="18" class="lang-bash editor-colors"><div style="color:Black;background-color:White;"><pre>
$ ls -al /bin/bash
-rwxr-xr-x. 1 root root 1390096 May 16 18:01 /bin/bash
$ ls -al /usr/bin/bash
-rwxr-xr-x. 1 root root 1390096 May 16 18:01 /usr/bin/bash
</pre></div>
</div>
```

Thanks for your hard work so far. 🙏🏼 

### 1/2 NPP Default Light Mode & corresponding Markdown Panel colors:

![image](https://user-images.githubusercontent.com/7158596/209774876-2ca152f8-f2a5-418c-8bb2-9281ef53f244.png)

### 2/2 NPP Default Dark Mode & corresponding Markdown Panel colors:

![image](https://user-images.githubusercontent.com/7158596/209774927-99b5a2c8-2965-4b31-b733-09c3b7d677f4.png)

**Comment by mohzy83** [comment]
Hi,
you are right, hardcoded colors are bad.
There should be proper class names, to be able to add a dark style (css).
I found the reason for this issue.
There are two html formatter classes available in the syntax highlighting lib ColorCode.
The one currenty used is: https://github.com/RichardSlater/ColorCodePortable/blob/master/ColorCode/Formatting/HtmlFormatter.cs -> which outputs hardcoded colors.
The other one outputs proper class names: https://github.com/RichardSlater/ColorCodePortable/blob/master/ColorCode/Formatting/HtmlClassFormatter.cs
I will change the implementation to use the class HtmlClassFormatter.cs .
But that also requires add the classes to `style.css` and `style-dark.css`.

**Comment by mohzy83** [comment]
fixed in release 0.7.2
